invenio/modules/access/local_config.py

```diff
@@ -558,7 +558,8 @@
                ('claimpaper_change_others_data', 'Change data of any person ID', '', 'no'),
                ('runbibtasklet', 'run BibTaskLet', '', 'no'),
                ('cfgbibsched', 'configure BibSched', '', 'no'),
-               ('runinfomanager', 'run Info Space Manager', '', 'no')
+               ('runinfomanager', 'run Info Space Manager', '', 'no'),
+               ('usedeposit', 'use deposit type', 'type', 'yes'),
```

Comment:
this line is not needed if you define your own action in https://github.com/inveniosoftware/invenio/pull/2906/files#diff-1917d7fa51639dedef09e17079a8a7cdR25